Input/Input.module.less

```diff
@@ -5,6 +5,8 @@
 @import "../styles/variables.less";
 
 .decorator {
+	align-items: center;
```

Comment:
For the next reviewer: this was needed to solve vertical alignment of text and icons on some skins: 
![image](https://user-images.githubusercontent.com/63335068/122236594-52544400-cec7-11eb-86aa-c11dd6901ee0.png)